ReportingServicesTools/Functions/CatalogItems/Write-RsCatalogItem.ps1

```diff
@@ -76,12 +76,25 @@ function Write-RsCatalogItem
         foreach ($item in $Path)
         {
             #region Manage Paths
-            if (!(Test-Path $item))
+            if ($item.StartsWith("\\"))
             {
-                throw "No item found at the specified path: $item!"
+                $EntirePath = ConvertTo-UNCFilePath $item
+
+                if (!(Test-Path $EntirePath))
+                {
+                    throw "No item found at the specified path: $EntirePath!"
+                }
+            }
+            else
+            {
+                if (!(Test-Path $item))
```

Comment:
What if you do only 1 Test-Path check and in if/else determine which value you want to use for $EntirePath?